packages/components/src/templates/next/components/internal/Navbar/MobileNavMenu/MobileNavItemAccordion.tsx

```diff
@@ -27,12 +36,81 @@ const mobileItemStyles = tv({
       true: {
         container: "pb-6",
         chevron: "-rotate-180",
+        menuItemsContainer: "flex flex-col gap-6",
+      },
+      false: {
+        menuItemsContainer: "hidden",
+      },
+    },
+    itemType: {
+      default: {
+        item: "justify-between gap-6",
+      },
+      parentItem: {
+        item: "gap-1",
+        nestedItem: "flex items-center gap-1",
       },
     },
+    withVerticalPadding: {
+      true: {
+        item: "py-1",
+      },
+    },
+  },
+  defaultVariants: {
+    itemType: "default",
   },
 })
 
-const { item, chevron, container, nestedItem, sublist } = mobileItemStyles()
+const { item, chevron, container, nestedItem, sublist, menuItemsContainer } =
+  mobileItemStyles()
+
+const ParentItemLink = ({
+  name,
+  referenceLinkHref,
+  isExternal,
+  LinkComponent,
+}: ParentItemLinkProps) => {
+  // This is a hack to ensure that the rightArrow is always at the end of the last word even on smaller screens
+  const words = name.trim().split(" ")
+  const allButLastWord = words.slice(0, -1).join(" ")
+  const lastWord = words[words.length - 1]
+
+  return (
+    <div
+      className={item({
+        itemType: "parentItem",
+        withVerticalPadding: true,
+      })}
+    >
+      <Link
+        LinkComponent={LinkComponent}
+        isExternal={isExternalUrl(referenceLinkHref)}
+        showExternalIcon={isExternalUrl(referenceLinkHref)}
+        isWithFocusVisibleHighlight
+        href={referenceLinkHref}
+        className={nestedItem({
+          className: `group/parent-item ${focusVisibleHighlight()}`,
+          itemType: "parentItem",
+        })}
+      >
+        <span className="row-gap-0 flex flex-row flex-wrap items-baseline gap-1">
+          Pages in
+          <span className="prose-headline-base-medium">{allButLastWord}</span>
+          <span className="prose-headline-base-medium flex flex-row items-center gap-1">
+            {lastWord}
+            {!isExternal && (
+              <BiRightArrowAlt
+                aria-hidden
+                className="text-[1rem] transition ease-in group-hover/parent-item:translate-x-1"
```

Comment:
[question] i thuoght the group hover syntax had to be `group-hover` (from the [docs](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state))

Reply:
ah yes but since there's 2 group being used here, we use `group-hover/{name}` to specify which group it belongs to (https://tailwindcss.com/docs/hover-focus-and-other-states#differentiating-nested-groups)